src/modules/enrollment/components/EnrollmentQuickActions.tsx

```diff
@@ -28,9 +27,16 @@ const EnrollmentQuickActions = ({
       clientId: enrollment.client.id,
     });
 
-  const canRecordService =
-    enabledFeatures.includes(DataCollectionFeatureRole.Service) &&
-    enrollment.access.canEditEnrollments;
+  const canRecordService = useMemo(() => {
+    const serviceFeature = enrollment.dataCollectionFeatures.find(
+      (f) => f.role === DataCollectionFeatureRole.Service
+    );
```

Comment:
Is there a reason the hook isn't used here?

Reply:
No reason, it can be used here :)